spec/features/user_browses_loan_requests_spec.rb

```diff
@@ -1,22 +1,32 @@
 require "rails_helper"
 
-RSpec.feature "user browses items" do
-  let(:create_item) { LoanRequest.create( ) }
+RSpec.feature "unauthenticated user browses loan requests" do
+  let(:create_item) { LoanRequest.create(title: "Farm Tools",
+                                         description: "help out with the farm tools",
+                                         amount: "$100.00",
+                                         requested_by_date: "2015-06-01",
+                                         repayment_begin_date: "2015-12-01",
+                                         repayment_rate: "Monthly")
+                    }
```

Comment:
`end` at 10, 20 is not aligned with `let(:create_item) { LoanRequest.create(title: "Farm Tools",` at 4, 2